lib/turnip/placeholder.rb

```diff
@@ -19,6 +20,10 @@ def find(name)
         placeholders[name] or default
       end
 
+      def clear
```

Comment:
IMHO adding a new public API to eliminate warnings doesn't make sense to me.
How about define the `clear` method in a Refinement and using it in the test instead?

@gongo What do you think?

```rb
# spec/support/clear_place_holders
module ClearPlaceHolders
  refine(Turnip::Placeholder) do
    def clear
      placeholders.clear
    end
  end
end

# spec/placeholder_spec.rb
using ClearPlaceHolders

describe Turnip::Placeholder do
  after do
    described_class.clear
  end
  # ...
end
```

Reply:
@hanachin Thanks! I think it's good
@graemeboyd I don't think it's good to make it Public API if `Placeholder.clear` just code for testing.